shapash/explainer/smart_explainer.py

```diff
@@ -191,17 +190,16 @@ def __init__(
                 """
             )
         self.model = model
+        self.preprocessing = preprocessing
+        self.backend_name = None
```

Comment:
if you create an attribute can you add in docstrings attributes

Reply:
done